### PR TITLE
Created astronomy dockerfile.

### LINKED ISSUE #190: Dockerise astronomy pipeline

### PR DESCRIPTION
**Pull request by olliejth**
Created dockerfile in line with the format used for the other repository directories.

closes #190

### CODE REVIEW

File: astronomy-pipeline/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+COPY requirements.txt .
+RUN pip install -r requirements.txt 
+
+COPY astronomy_load.py .
+COPY astronomy_extract.py .
+COPY astronomy_transform.py .
+COPY astronomy_extract_functions.py .
+COPY astronomy_transform_functions.py .
+COPY astronomy_lambda.py .
```
Comment (AlexOBarnes) [inline astronomy-pipeline/Dockerfile:13]: Thanks for updating this!

File: astronomy-pipeline/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+CMD [ "astronomy_lambda.lambda_handler" ]
```
Comment (AlexOBarnes) [inline astronomy-pipeline/Dockerfile:15]: Much more inline with the naming conventions

**Review by AlexOBarnes — APPROVED** [review]

File: astronomy-pipeline/Dockerfile (hunk 2 above)
Comment (MurasatoNaoya) [reply astronomy-pipeline/Dockerfile:15]: Agreed, follows Dockfile industry styling. Boo ya!

**Review by MurasatoNaoya — APPROVED** [review]
LGTM.

**Pull request merged**